src/product-card.css

```diff
@@ -0,0 +1,94 @@
+.container {
+  width: 198px;
+  box-sizing: border-box;
+  padding: 0 16px;
+}
+
+.container--card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.container__picture {
+  font-family: "Roboto", sans-serif;
+  width: 160px;
+  margin-left: 3px;
+  margin-right: 19px;
+  margin-top: 32px;
+  margin-bottom: 40px;
+}
+
+.container__link-first {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  text-decoration: none;
+}
+
+.productCode {
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.container__price {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  margin-bottom: 20px;
+}
```

Comment:
try to set only bottom or top margins
it is good approach to avoid `margin collapse`